sass/_OmniboxResultList.scss

```diff
@@ -2,5 +2,12 @@
 	&.coveo-omnibox-selectable {
 	  padding: 10px 10px 10px 36px;
 	  margin: 0;
+		box-sizing: border-box;
+	}
+	&.coveo-omnibox-result-list-element.coveo-omnibox-selectable.coveo-omnibox-selected {
```

Comment:
Alignment is weird. Rest of css is 2 spaces.

Also, does the selector needs to be *this* precise ?

The simpler the selector is, the easier it gets for customization.

Otherwise, someone needs to write an even more precise rule to add his own customization.

Always try to use the least precise selector you possibly can that still does the job.